tools/README-generator/make_readme.py

```diff
@@ -53,6 +59,12 @@ def generate_READMEs(app_path: str):
         else:
             disclaimer = None
 
+        # TODO: Add url to the documentation... and actually create that documentation :D
+        antifeatures = [antifeatures_list[a] for a in from_catalog.get('antifeatures', [])]
+        for antifeature in antifeatures:
+            antifeature['title'] = antifeature['title'].get(lang_suffix, 'en')
+            antifeature['description'] = antifeature['description'].get(lang_suffix, 'en')
```

Comment:
Wild thought: could we allow to define one's own description in the manifest, if one wants to be more specific than the generic descriptions in `antifeatures.yaml`?

Reply:
My guess was that more specific details could go under the "Disclaimer" section, and this was a more generic warnings about that "type" of anti-feature, not the app implementation of the anti-feature.
But since this is the app README, that could make sense to have an additional description from the manifest, and hoping that it's not confusing for the one installing the app.